docs/cache/manage/limits.md

```diff
@@ -27,61 +27,63 @@ The limits on this page are soft limits that can be altered unless specifically
-| SetAddElements           | Number of elements in request                                   |
-| SetFetch                 | Number of elements in response, or 1 if set is not found        |
-| SetRemoveElement         | 1                                                               |
-| SetRemoveElements        | Number of elements in request                                   |
-| SetContainsElement       | 1                                                               |
-| SetContainsElements      | Number of elements in request                                   |
-| SetLength                | 1                                                               |
-| SortedSetPutElement      | 1                                                               |
-| SortedSetPutElements     | Number of elements in request                                   |
-| SortedSetFetchByRank     | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetFetchByScore    | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetGetScore        | 1                                                               |
-| SortedSetGetScores       | Number of elements in request                                   |
-| SortedSetRemoveElement   | 1                                                               |
-| SortedSetRemoveElements  | Number of elements in request                                   |
-| SortedSetGetRank         | 1                                                               |
-| SortedSetIncrementScore  | 1                                                               |
-| SortedSetLength          | 1                                                               |
-| SortedSetLengthByScore   | 1                                                               |
+| API Name                 | Batch API | Operations                                                        |
+| ------------------------ | ----      | ------------                                                      |
+| Set                      |           | 1                                                                 |
+| Get                      |           | 1                                                                 |
+| Delete                   |           | 1                                                                 |
+| Increment                |           | 1                                                                 |
+| Ping                     |           | 1                                                                 |
+| ItemGetType              |           | 1                                                                 |
+| KeyExists                |           | 1                                                                 |
+| KeysExist                | X         | Number of keys in request/2                                       |
```

Comment:
This isn't 100% accurate... really it's `ceil(Number of keys in request/2)` but I'm not sure how we want to represent that in this table?

Reply:
that's probably close enough imo